.github/workflows/requirements-lint.txt

```diff
@@ -0,0 +1,67 @@
+#
```

Comment:
AFAICT this is like a lock file. Should we perhaps use the `Pipfile` and `Pipfile.lock` formats instead?

https://pipenv.pypa.io/en/latest/pipfile.html

Reply:
That's right, it is a lock file. I tried pipenv, but you first have to install it into the actions runner. Even actions/setp-python doesn't actually install pipenv. So I think we may as well continue using the requirements.txt.